project-card.js

```diff
@@ -0,0 +1,126 @@
+const project = [
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    Featured_image: '/img/card-bg1.png',
+    Technologies: ['HTML', 'css', 'ruby'],
+    liveVersion: 'https://enning94.github.io/Personal-portfolio/',
+    sourceLink: 'https://github.com/Enning94/Personal-portfolio',
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    Featured_image: '/img/card-bg1.png',
+    Technologies: ['HTML', 'css', 'ruby'],
+    liveVersion: 'https://enning94.github.io/Personal-portfolio/',
+    sourceLink: 'https://github.com/Enning94/Personal-portfolio',
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    Featured_image: '/img/card-bg1.png',
+    Technologies: ['HTML', 'css', 'ruby'],
+    liveVersion: 'https://enning94.github.io/Personal-portfolio/',
+    sourceLink: 'https://github.com/Enning94/Personal-portfolio',
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    Featured_image: '/img/card-bg1.png',
+    Technologies: ['HTML', 'css', 'ruby'],
+    liveVersion: 'https://enning94.github.io/Personal-portfolio/',
+    sourceLink: 'https://github.com/Enning94/Personal-portfolio',
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    Featured_image: '/img/card-bg1.png',
+    Technologies: ['HTML', 'css', 'ruby'],
+    liveVersion: 'https://enning94.github.io/Personal-portfolio/',
+    sourceLink: 'https://github.com/Enning94/Personal-portfolio',
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    Featured_image: '/img/card-bg1.png',
+    Technologies: ['HTML', 'css', 'ruby'],
+    liveVersion: 'https://enning94.github.io/Personal-portfolio/',
+    sourceLink: 'https://github.com/Enning94/Personal-portfolio',
+  },
```

Comment:
- [ ] Your popup is working properly but it effect the design as the design has more spacing on right kindly fix it
### Your design
![his](https://user-images.githubusercontent.com/81727988/180445606-d141c8ad-1462-4a1e-af4e-4599d6e0ede1.png)